dashboard/src/App.js

```diff
@@ -1,26 +1,65 @@
-import React from 'react';
+import React, { useState } from 'react';
 import logo from './logo.svg';
 import './App.css';
 
-//component imports
-import Index from "../src/components/Index";
+/*
+class App extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      items = []
+    }
+  }
+
+  render() {
+    const { items } = this.state;
+    return <>
+      <input> </input>
+      <h1> gg </h1>
+      <ul>
+        {items.map(item => <li>{item} </li>)}
+      </ul>
+    </>
+  }
+}*/
 
+//component imports
+import MainPage from "./components/MainPage";
+import Banner from "./components/Banner";
+import HowTo from "./components/HowTo";
+import Leaderboard from "./components/Leaderboard";
```

Comment:
something to consider when you start importing a lot of files from the same folder is whether to create an index.js that exports the components from the folder as a module. Then you can write import { MainPage, Banner ... } from './components' which can be a lot cleaner, especially if you're importing like 8 things from the same folder. Here's an example of what I'm talking about https://www.digitalocean.com/community/tutorials/react-index-js-public-interfaces